config/environments/production.rb

```diff
@@ -70,19 +79,14 @@
   # Don't log any deprecations.
   config.active_support.report_deprecations = false
 
-  # Use default logging formatter so that PID and timestamp are not suppressed.
-  config.log_formatter = ::Logger::Formatter.new
-
-  # Use a different logger for distributed setups.
-  # require "syslog/logger"
-  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
-
-  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
Should we be applying this env check to the new logger configuration?

Now I see it, I'm not 100% sure I did for manuals 🤔

Reply:
Hmm good spot 🧐 I'm not totally sure about this to be honest. My gut says we should.

Reply:
I've decided to include this logic to be on the safe side.